extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -253,9 +253,8 @@ class AmpApesterMedia extends AMP.BaseElement {
    * @return {!Element}
    */
   constructLoaderImg_() {
-    const img = this.element.ownerDocument.createElement('amp-img');
+    const img = this.element.ownerDocument.createElement('img');
     img.setAttribute('src', this.loaderUrl_);
```

Comment:
This can pretty easily cause a CLS if the img isn't sized exactly 100x100. We need to use `style="width: 100px; height: 100px"`